events/lunch-and-learn.md

```diff
@@ -56,57 +67,67 @@ Here are some examples of internal presentations:
 
 ---
 
-You're probably seeing this as an external developer to Artsy, who has been invited to talk.
+You're probably seeing this as an external developer to Artsy, who has been
+invited to talk.
 
 ## Format
 
-The format is definitely not set in stone, however these are what has worked for us internally and for past
-contributors.
+The format is definitely not set in stone, however these are what has worked for
+us internally and for past contributors.
 
 #### Presentation
 
-Traditional conference/meetup style: You bring slides, we watch. You talk, we listen. You finish, we ask questions.
+Traditional conference/meetup style: You bring slides, we watch. You talk, we
+listen. You finish, we ask questions.
 
-We don't expect conference/meetup-level presentation or practice. A lack of polish is okay. This is a great
-opportunity to present new talks ideas.
+We don't expect conference/meetup-level presentation or practice. A lack of
+polish is okay. This is a great opportunity to present new talks ideas.
 
 #### Company Walkthrough
 
-More informal, you can come with other speakers and talk about the ways in which your company works. Covering your
-tech stack, the decisions behind them. With the presentation mainly being a series of demos, and showing code
-architecture.
+More informal, you can come with other speakers and talk about the ways in which
+your company works. Covering your tech stack, the decisions behind them. With
+the presentation mainly being a series of demos, and showing code architecture.
 
 ## Schedule
 
-- Arrive at Artsy for 12:15pm (we're [401 Broadway, 26th Floor, 10013][401] - Canal St is the closest station)
-- Our receptionist on 26 will get in touch with whoever you've been in contact with, probably [Orta][] or [Ash][]
+- Arrive at Artsy by 12:15pm ([401 Broadway, 26th Floor, 10013][401] - Canal St
+  is the closest station)
+- Our receptionist on 26 will get in touch with whoever you've been in contact
+  with, probably [Orta][] or [Ash][]
```

Comment:
```suggestion
  with, probably [Ash][]
```

Open to someone else stepping up as a second L&L NYC point-person. @anandaroop would you be interested?

Reply:
Yeah let’s do it 💪🏽